### ISSUE TITLE
When turn a outcome into a "Small" the text disappears - hard to see what you are supposed to do with it

### ISSUE THREAD

**Issue by mitra42** [lightningrodlabs/acorn#222]
(Note, I'm a new user of Acorn, so I may be misunderstanding things (and its possible I'll use the wrong terms as they are pretty Acorn-unique), but in this case the feedback may be useful in terms of areas of confusion for new users !

STR - create an Outcome "Called a bunch of leads"
Create a sub-outcom of it. 
and give it a name "Call foo"
Click into it to open up the dialog for that outcome 
Set Scope to small and close dialog with the X
It changes color and has the Leaf icon (good)
the text "Call foo" disappears, and only appears when you hover over the leaf (as a blue box over the top)
There is now no way to see what list of small outcomes make up "Called a bunch of leads" 
I've tried adding tasks, or description to "Call foo" but the only way I can see anything about this Small Scope Outcome  is to hover over it.

**Comment by TechSupport26** [comment]
Hi mitra42. You can scroll/zoom in to see the text on small scope cards.
This is a recent feature (introduced a few versions ago)
I think goes along well with the new design that has been implemented. Have a look at old images used when all the cards are the same size.

I hope this answers your question.

**Comment by mitra42** [comment]
OK thanks - i can confirm that works - this is seriously non obvious, I was fighting the UX trying to figure out why this had rendered the map unusable. Why not just put the text in the card in a smaller font, then scrolling in becomes an obvious thing to do ?

**Comment by TechSupport26** [comment]
Puting the text in a smaller font was done in previous versions. (IIRC it was like that from the begining)

**Comment by jakobwinter** [comment]
> Why not just put the text in the card in a smaller font, then scrolling in becomes an obvious thing to do ?

Yeah, having the font scale with the tile when zooming would be nice.

**Comment by Connoropolous** [comment]
Tagging @pegahvaezi in on this one, as it comes down to UX.
